src/Button/BorderlessButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {
+  FunctionComponent, ReactElement, useContext, ReactNode, MouseEventHandler,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+import { StyledButton } from './StyledButton';
+
+export interface BorderlessButtonProps {
+  /** Text or components to be displayed on the button */
+  children?: ReactNode;
+  /** Function to call on click event */
+  clickHandler: MouseEventHandler;
+  /** If true, button won't fire */
+  disabled?: boolean;
+  /** Allows you to pass in style properties for the element */
+  theme?: object;
+  /** Allows you to pass in a color property from the COLOR enum */
+  color?: COLOR;
+}
+
+enum COLOR {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+const StyledBorderlessButton = styled(StyledButton)`
+  background: transparent;
+  border-style: none;
+  color: ${({ theme, color }): string => (theme.color.text[color])};
+  &:hover {
+    background: transparent;
+    color: ${({ theme, color }): string => (theme.color.text[color])};
```

Comment:
I think we need to specify some kind of visual cue for the hover state to help users realize this is a button. Not 100% sure that would look like though. Can you do some research into how other frameworks/etc. handle that kind of UX? Or run it by Vittorio to see if he has ideas?

Reply:
Ok looking into this

Reply:
We decided to add a `variant` prop and set the default text color of the BorderlessButton to `theme.color.background[variant].medium` and to `theme.color.background[variant].dark` on hover.